spec/nepali_calendar_spec.rb

```diff
@@ -196,5 +196,19 @@
       fiscal_year = NepaliCalendar::FiscalYear.current_fiscal_year
       expect(fiscal_year.to_s).to eq('7879')
     end
+
+    it 'returns the next fiscal year' do
+      fiscal_year = NepaliCalendar::FiscalYear.new(78, 79).next
+      expect(fiscal_year.to_s).to eq('7980')
+    end
+
+    it 'returns the list of fiscal years in AD' do
```

Comment:
fiscal years in ad?
you mean fiscal year since a date in ad?

Reply:
It meant fiscal years with start date and end date in AD, this has been updated.